nio/events/account_data.py

```diff
@@ -67,6 +69,30 @@ def from_dict(cls, event_dict):
         )
 
 
+@attr.s
+class TagEvent(AccountDataEvent):
+    """Informs the client tags of a room. Room tags may include
+        - m.favourite for favourite rooms
+        - m.lowpriority for low priority room
+       A tag may have an optinal order float 0 <= order <=1 for position
+       this room to other rooms.
+
+    Attributes:
+        tags (Dict[string, Optional[Dict[string, float]]): The tag dictionary.
```

Comment:
`tags (Dict[str, Optional[Dict[str, float]]]): The tags of the room and their contents.`

The Python type is `str`, not `string`.